### ISSUE TITLE
config.eager_load_paths cannot be a glob pattern

### ISSUE THREAD

**Issue by fxn** [owen2345/camaleon-cms#895]
The engine is setting a glob pattern as eager load paths:

```
app.config.eager_load_paths += %W(#{app.config.root}/app/apps/**/)
```

but `config.eager_load_paths` has to be an array of paths.

This prevents eager loading in Rails 6. One error that you may get is

```
No such file to load -- es/camaleon_first/main_helper.rb
```

**Comment by brian-kephart** [comment]
@owen2345 I made a branch to address this issue, but there are a couple tests failing on Ruby 2.3. Details here: https://travis-ci.com/brian-kephart/camaleon-cms/jobs/195753567

I think I've seen similar failures in the past, but you were the one who solved them. Do you recall anything about this type of test failure? I'd like to get the tests passing before submitting a PR.

**Comment by brian-kephart** [comment]
@owen2345 this issue was fixed by #903 and can be closed.
